src/engine/enginemaster.cpp

```diff
@@ -97,6 +100,13 @@ EngineMaster::EngineMaster(UserSettingsPointer pConfig,
 
     m_pMasterDelay = new EngineDelay(group, ConfigKey(group, "delay"));
     m_pHeadDelay = new EngineDelay(group, ConfigKey(group, "headDelay"));
```

Comment:
I am unsure what is required for timecode if anything. I suspect it is as difficult to scratch on beat without input latency compensation as it is difficult to count into a microphone on beat without latency compensation. I would think that the deck's buffer would need to have the input latency compensation applied before applying the manipulation from timecode. I have not looked too closely at the timecode processing code, but from a quick look through EngineBuffer::process it looks like this would require a lot of complicated refactoring to implement. Let's leave that for another PR.